packages/expo-camera/android/src/main/java/expo/modules/camera/tasks/ResolveTakenPictureAsyncTask.java

```diff
@@ -66,7 +66,7 @@ public ResolveTakenPictureAsyncTask(Bitmap bitmap, Promise promise, Map<String,
   }
 
   private int getQuality() {
-    return ((Number) mOptions.get(QUALITY_KEY)).intValue() * 100;
+    return (int)(((double) mOptions.get(QUALITY_KEY)) * 100);
```

Comment:
Is it possible that `mOptions.get(QUALITY_KEY)` will be `null`? Maybe we could handle this case here (just to be safe) and fallback to `1`? This would also make this function a little bit longer with space for more explanatory variables.